rfcs/005-group-stats.md

```diff
@@ -0,0 +1,79 @@
+# Summary
+A page at the course-instance level for group statistics. 
+
+# Basic Example Workflow
+An instructor with appropriate access uploads a reference (i.e. desired) groups assignment for their class. PrairieLearn will display group-level statistics across assessments in a given course instance. Instructor can then decide to adjust filters and parameters to get desired statistics and be able to recompute and download. 
+
+*Group Participation* 
+
+Filters: [Exclude Assignment 2]
+
+| GroupName | Student IDs | Assessment 1 | ... | Assessment N|
+|-----------|-------------|--------------|-----|-------------|
+| GroupA    | Student 1   |      Y       | ... |      Y      |
+|           | Student 2   |      X       | ... |      Y      |
+|           | Student 3   |      Y       | ... |      Y      |
+|           | Student 4   |      Y       | ... |      Y      |
+
+(In this example, Student 2 did not work with their assigned group on assessment 1.)
+
+*Role Participation*
+
+Filters: [Include student data from any group]
+
+| Student ID | Role 1 | Role 2 | Role 3 | Role 4 |
+|------------|--------|--------|--------|--------|
+| Student 1  |    3   |   1    |   1    |   1    |
+| Student 2  |    1   |   1    |   1    |   1    |
+| Student 3  |    2   |   1    |   2    |   1    |
+| Student 4  |    0   |   2    |   1    |   2    |
+
+*Submission Count* 
```

Comment:
As you call out elsewhere, I can imagine this being particularly useful e.g. for identifying students who are doing ~100% or ~0% of the work!